src/components/pages/ProblemDetailedPage/index.tsx

```diff
@@ -45,15 +52,45 @@ const ProblemDetailedPage: FC = () => {
   const formattedDate =
     problemDetails.createdAt && formatDate(problemDetails.createdAt);
 
+  const printRef = useRef<HTMLDivElement>(null);
+
+  const handleDownloadPdf = async () => {
+    const element = printRef.current as HTMLDivElement;
+
+    const canvas = await html2canvas(element);
+    const imgData = canvas.toDataURL("image/png");
+    const pdf = new jsPDF("p", "mm");
+
+    let position = 0;
+
+    const imgProperties = pdf.getImageProperties(imgData);
+    const pdfWidth = pdf.internal.pageSize.getWidth() - 20;
```

Comment:
Can't we just use `width` instead of `getWidth()`? : >

Reply:
I don't think so -> https://github.com/parallax/jsPDF/issues/1832

Reply:
```
    const pdfWidth = pdf.internal.pageSize.width - 20;
```
Works for me as well, I noticed that, as I hovered over to see the interface of the `pdf.internal` ;)

Reply:
It this any particular reason for changing it? I don't mind to change it, i am just curious. :)

Reply:
If there is just a key that gets what I won't I would use it over having to invoke the function that gets the same value, it just looks a bit simpler for me and doesn't make me think, opposed to `getWidth()` ;) But you can treat it as a neat pick, wouldn't "request change" for that for sure :D